src/ngTouch/swipe.js

```diff
@@ -138,12 +142,16 @@ ngTouch.factory('$swipe', [function() {
 
         lastPos = coords;
 
+		//set the buffer size to custom or to the default of 10
+		if (!angular.isNumber(verticalBufferSize)) {
+			verticalBufferSize=MOVE_BUFFER_RADIUS;
+		}
```

Comment:
Indentation is off.

Also, we shouldn't check `verticalBuffrSize` on each call. Let's set it once at the top of the `bind()` method.